### PR TITLE
UG about this guide

### PR DESCRIPTION
**Pull request by lingshanng**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -15,10 +15,23 @@ title: User Guide
 TAB comes with a clean Graphical User Interface (GUI) while optimised for user interaction via a CLI (Command Line Interface).
 With TAB, you can effortlessly manage your students' contact details faster than a typical mouse/GUI driven app.
 
+## About This Guide
+
+This user guide is intended for any user of TAB. It provides instructions for installing TAB, and gives a detailed description of each feature available in TAB.
```
Comment (eeliana) [inline docs/UserGuide.md:20]:
```suggestion
This user guide is intended for any user of TAB. It provides installation instructions to help you get started with using TAB, and gives detailed description of the features available in TAB.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ title: User Guide
...
+
+**Navigation**
+
+In the [Table of Contents](), each item listed is a link and you can click to go directly to that section in the guide.
+
+**Conventions Used**
+
+* `text` : denotes a command to be entered into the command box in TAB.
+* [text](#about-this-guide) : denotes links to other parts of the document or links to be opened in the browser.
```
[the remark this answers is not in the document]
Comment (lingshanng) [reply docs/UserGuide.md:29]: @Xiaoyunnn i'll change mine

File: docs/UserGuide.md [hunk 3]
```diff
@@ -15,10 +15,24 @@ title: User Guide
 TAB comes with a clean Graphical User Interface (GUI) while optimised for user interaction via a CLI (Command Line Interface).
 With TAB, you can effortlessly manage your students' contact details faster than a typical mouse/GUI driven app.
 
+## About This Guide
+
+This user guide is intended for any user of TAB. It provides installation instructions to help get you started with using TAB, and gives detailed descriptions of the features available in TAB.
```
Comment (eeliana) [inline docs/UserGuide.md:20]: I think the plural for description is also description

File: docs/UserGuide.md [hunk 4]
```diff
@@ -31,14 +45,14 @@ Here is a [website](https://docs.oracle.com/en/java/javase/17/install/overview-j
...
-2. Download the latest `TAB.jar` from [here](https://github.com/AY2122S1-CS2103T-F13-3/tp/releases).
+2. Download the latest **TAB.jar** from [here](https://github.com/AY2122S1-CS2103T-F13-3/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your TAB.
 
 4. Double-click the file to start the app. The window similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press <kbd>ENTER</kbd> to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```
Comment (eeliana) [inline docs/UserGuide.md:55]:
```suggestion
5. Type the command in the command box and press <kbd>ENTER</kbd> to execute it. e.g. typing **`help`** and pressing <kbd>ENTER</kbd> will open the help window.<br>
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -15,10 +15,24 @@ title: User Guide
...
+This user guide is intended for any user of TAB. It provides installation instructions to help get you started with using TAB and describes in detail the features available in TAB.
+
+**Navigation**
+
+In the [Table of Contents](), each item listed is a link which you can click on to go directly to that section in the guide.
+
+**Conventions Used**
+
+* `text` : denotes a command to be entered into the command box in TAB.
+* <kbd>text</kbd> : denotes a keyboard input.
```
Comment (Xiaoyunnn) [inline docs/UserGuide.md:29]: keyboard or button?

**Pull request merged**